qutebrowser/javascript/webelem.js

```diff
@@ -40,20 +40,10 @@ window._qutebrowser.webelem = (function() {
     const funcs = {};
     const elements = [];
 
-    function get_frame_offset(frame) {
```

Comment:
Hmm, not sure if all this stuff should be moved as well, seeing that it's only used in `webelem.py`.